pkg/scheduler/preemption/preemption.go

```diff
@@ -368,9 +371,8 @@ func parseStrategies(s []config.PreemptionStrategy) []fairsharing.Strategy {
 // and returns (fits, targets, retryCandidates) retryCandidates may be
 // used if rule S2-b is configured.
 func runFirstFsStrategy(preemptionCtx *preemptionCtx, candidates []*workload.Info, strategy fairsharing.Strategy) (bool, []*Target, []*workload.Info) {
-	requests := preemptionCtx.requests
 	cqHeap := cqHeapFromCandidates(candidates, false, preemptionCtx.snapshot)
-	preemptorNewShare := fairsharing.PreemptorNewShare(preemptionCtx.preemptorCQ.DominantResourceShareWith(requests))
+	preemptorNewShare := fairsharing.PreemptorNewShare(preemptionCtx.preemptorCQ.DominantResourceShare())
```

Comment:
Just a comment, I see this is the same since we call `preemptionCtx.preemptorCQ.AddUsage(preemptionCtx.workloadUsage)` before entering `runFirstFsStrategy`. Nice.